contracts/BosonRouter.sol

```diff
@@ -129,18 +148,19 @@ contract BosonRouter is
     }
 
     /**
-     * @notice Issuer/Seller offers promises as supply tokens and needs to escrow the deposit
-        @param metadata metadata which is required for creation of a voucher
-        Metadata array is used as in some scenarios we need several more params, as we need to recover 
-        owner address in order to permit the contract to transfer funds on his behalf. 
-        Since the params get too many, we end up in situation that the stack is too deep.
-        
-        uint256 _validFrom = metadata[0];
-        uint256 _validTo = metadata[1];
-        uint256 _price = metadata[2];
-        uint256 _depositSe = metadata[3];
-        uint256 _depositBu = metadata[4];
-        uint256 _quantity = metadata[5];
+     * @notice Issuer/Seller offers promise as supply token and needs to escrow the deposit. A supply token is
+     * also known as a voucher set. Payment and deposits are specified in ETH.
+     * @param metadata metadata which is required for creation of a voucher set
+     * Metadata array is used for consistency across the permutations of similar functions.
+     * Some functions require other parameters, and the number of parameters causes sa tack too deep error.
```

Comment:
"sa tack" -> "stack".
The same mistake is repeated multiple times below, so I won't tag it everywhere

Reply:
Included in next commit.